po/be.po

```diff
@@ -3405,7 +3405,7 @@ msgstr ""
 msgid "Toggle Insert/Overwrite mode"
```

Comment:
Yes, I tried to restore the not changed translation files to the original state but that did not work. The only really changed file is ```po/de.po```.

Maybe I should place a new fresh PR is all points are sorted out to keep the other translation files unchanged?